### PR TITLE
Fix brittle test

### PR DESCRIPTION
**Pull request by magopian**
The main change here is to make sure the created date is set, or it might be
the same for the two versions for addon `Nominated Two`, and thus returned in
the "wrong" order.

### CODE REVIEW

File: apps/editors/tests/test_views.py [hunk 1]
```diff
@@ -975,31 +975,33 @@ def test_breadcrumbs(self):
...
-        f = v2.all_files[0]
+        version1 = self.addons['Nominated One'].versions.all()[0]
+        version2 = self.addons['Nominated Two'].versions.all()[0]
+        file_ = version2.files.get()
 
-        original_nomination = v2.nomination
-        v2.nomination = v2.nomination - timedelta(days=1)
-        v2.save()
+        # Versions are ordered by creation date, so make sure they're set.
+        past = datetime.now() - timedelta(days=1)
```
Comment (robhudson) [inline apps/editors/tests/test_views.py]: You can use `self.days_ago(1)`

**Comment by robhudson** [comment]
r+wc. I fixed a lot of these on zamboni in the past.

File: apps/editors/tests/test_views.py (hunk 1 above)
Comment (magopian) [inline apps/editors/tests/test_views.py]: Clever, didn't know about this helper, thanks!

File: apps/editors/tests/test_views.py [hunk 2]
```diff
@@ -975,31 +975,33 @@ def test_breadcrumbs(self):
...
 
-        # Create another version, v0.2.
-        v2.pk = None
-        v2.nomination = original_nomination
-        v2.version = '0.2'
-        v2.save()
+        # Create another version, v0.2, by "cloning" v0.1.
+        version2.pk = None
+        version2.version = '0.2'
+        future = datetime.now() - timedelta(seconds=1)
```
Comment (robhudson) [inline apps/editors/tests/test_views.py]: could have used `days_ago` here too. :smile:

Comment (robhudson) [inline apps/editors/tests/test_views.py]: oh, nvm, that's seconds

**Pull request merged**